staging/src/k8s.io/apimachinery/pkg/api/resource/quantity_test.go

```diff
@@ -1820,3 +1820,36 @@ func TestQuantityRoundtripCBOR(t *testing.T) {
 		}
 	}
 }
+
+func TestMilliChecked(t *testing.T) {
+	for _, tc := range []struct {
+		name        string
+		q           *Quantity
+		expectError bool
+	}{
```

Comment:
TODO: pretty limited test cases, then again this method is fairly trivial and the point is to catch the expected overflow but not spuriously ...